### PR TITLE
build: Multiple requirements enhancements.

### PR DESCRIPTION
**Pull request by ashcrow**
- Added `python-etcd` to requirements.txt
- Licenses added to requirements.txt in an annotation like format.
- setup.py now handles comments on requirements lines.
#28 notes the need for `python-etcd`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def extract_names(filename):
 
 def extract_requirements(filename):
     with open(filename, 'r') as requirements_file:
-        return [x[:-1] for x in requirements_file.readlines()]
+        return [x[:-1].split(' #')[0] for x in requirements_file.readlines()]
```
Comment (mbarnes) [inline setup.py]: Not required, but you could just tokenize the line and return the first token.  Slightly simpler.

`return [x.split()[0] for x in requirements_file.readlines()]`

**Comment by ashcrow** [comment]
:arrow_up:

**Review by mbarnes — APPROVED** [review]

**Comment by mbarnes** [comment]
I'm gonna just ignore Red Hat CI for the moment.  Seems to be having its own issues.

**Comment by jlebon** [comment]
Yeah, that happens sometimes. :( till not sure what goes on, but it seems like a common issue with dnf in docker.

**Pull request merged**